### ISSUE TITLE
Ban Appeal #1002 V2

### ISSUE THREAD

**Issue by Daegonna** [ObliqueNET/Server#1019]
I apologise for the delay in the response, so i'll forgive the tone. 

In regards to your ban, you have broken few server rules, such as using AFK machines and griefing unclaimed bases. You were temporarily banned after your 2nd offence, which was also your final chance. And now you were permanently banned for griefing another unclaimed base. Its hard to argue that you unaware of the server rules, as you would've been told to read them at some point, or you should've read the rules if you were 'afraid of doing something wrong without knowing'. On these grounds, I am declining your appeal.

_Originally posted by @Katadreme in https://github.com/ObliqueNET/Server/issues/1002#issuecomment-783984057_

**Comment by Daegonna** [comment]
I never used AFK machines so that is defently not true. After i got warned and gave stuff back for taking stuff from a unclaimed base. I didn't do that on purpose either. They said i wasn't allowed to take stuff from unclaimed bases so i didn't. I did take some stuff from bases that were already distroyed. Didn't think that was an isseu coz than it is given back to nature right? And last u r not talking to a kid, but a mom that just wants to play a game with her kid.

**Comment by Daegonna** [comment]
P.S.
I'm really wondering why ppl keep saying i use afk machines coz i don't even know how 2 build 1.

**Comment by Rasgnarok** [comment]
I just want to clarify that your ban is **way** more about griefing bases than it is about AFK machines. You don't get banned for AFK machines, but you do get banned for stealing after being warned several times. Considering you were clearly told not to, and did it anyways, I am not sure why you are justifying doing it anyways.

**Comment by Rasgnarok** [comment]
I am leaving this open to see if you have anything else to say, otherwise I will be closing this by the next 24 hours.

**Comment by Daegonna** [comment]
Vetrn is the only 1 who actualy did talk 2 me and my son.  He said as far as  i know i shouldn't take stuff from unclaimed  bases, so i didn't do that anymore for as far as i know. When i got my temp ban, i asked why i got banned. I didn't get an answer other than just sit it out. My son asked about my ban he didn't get an answer. I kept asking what was the problem. And didn't get an answer. In my comment earlier i said i took stuff from a bese that was already distroyed for most part. But by then it is given back 2 nature right? Coz sinds my temp ban i didn't leave base much.

**Comment by Daegonna** [comment]
Could some1 b so kind 2 answer?

**Comment by Katadreme** [comment]
Lets break down your reasons for your ban:
To clarify about your afk machine. You weren't using a machine, but a strategy that avoids afk kicker which is staying afk whilst flying on a flying Pokemon - this being your Togekiss, which you were found to be on for some time. You were spoken to regarding this issue by PJLesure, who said you promised to not do anything like this again. Chance #1 gone. 

Then you were caught griefing / stealing from a unclaimed based, which lets solidify, is against the server rules. That includes stealing from unclaimed bases, even if they are as you say 'returning back to nature'. Regarding this grief, you were spoken to by Vetrnn, who says you had returned the items, apologised and assured this wouldn't happen again. Chance #2 gone. Technically out of chances now. 

You were then tempbanned for not responding to staff when they were trying to contact you about another less severe grief. You didn't respond to staff for more than 10minutes, hence resulting in the tempban, which answers your question about why you were tempbanned, which I also answered in the tempban appeal you had written, more than just 'sit it out'. Chance #3 gone after you were unbanned.

And then you were permanently banned as we had found another grief in an unclaimed base by you. So multiple chances given, promises made not to break the rules, and you were told to read the rules as well. You say that you wouldn't break rules on purpose, but after 3 instances of rule-breaking, kind of hard to prove that now, and saying you were unaware doesn't work either. If its not clear now why you were banned, not sure what I can tell you. And all the appeals you have written, including this one, have all been more 'I got banned but I don't think I did anything wrong' (despite me and other staff clarifying the reasons for your bans), than have been actual appeals. So yeah, your appeal has been declined. Again.
